safe/definitions.py

```diff
@@ -772,18 +763,18 @@
     'type': 'field',
     'description': tr('Attribute for the area name ')
 }
-area_type = {
+area_type_field = {
     'key': 'area_type_field',
     'name': tr('Type field'),
     'type': 'field',
     'description': tr('Attribute for the type of the area')
 }
 
-area_population_field = {
-    'key': 'area_population_field',
-    'name': tr('Population field'),
+area_id_field = {
+    'key': 'area_id_field',
+    'name': tr('Id field'),
     'type': 'field',
-    'description': tr('Attribute for the population on the area')
+    'description': tr('Attribute for the id on the area')
```

Comment:
Hi @timlinux if there are several Polygon, with the same Kelurahan Name, should we merge the result or we do it in aggregation? Currently, it will list all different id.